packages/@webex/plugin-meetings/src/locus-info/index.ts

```diff
@@ -217,18 +250,18 @@ export default class LocusInfo extends EventsScope {
    * @returns {undefined}
    * @memberof LocusInfo
    */
-  emitScoped(scope, eventName, args) {
+  emitScoped(scope: string, eventName: string, args: Array<any>) {
```

Comment:
`args: Array<any>` can be `args: any[]`

Reply:
Same as [this](https://github.com/webex/webex-js-sdk/pull/2508#discussion_r1033647085)